responses/configuration/set.go

```diff
@@ -8,26 +8,21 @@ import (
 
 //SetConfigResponse is for SMA to use when responding to an incoming request to PUT (i.e. UPDATE) a resource.
 type SetConfigResponse struct {
-	Code           string   `json:"code" yaml:"code,omitempty"`                     //HTTP code to return (e.g. 200, 500, etc.)
-	Description    string   `json:"description" yaml:"description,omitempty"`       //info allied with preceding HTTP code
-	ExpectedValues []string `json:"expectedValues" yaml:"expectedValues,omitempty"` //expected values to return
-	isValidated    bool     //internal member used for validation check
+	Success     bool   `json:"success"`               // HTTP success or failure
+	Description string `json:"description,omitempty"` // info allied with preceding HTTP code
+	isValidated bool   // internal member used for validation check
 }
 
 // Custom marshalling to make empty strings null
 func (sc SetConfigResponse) MarshalJSON() ([]byte, error) {
 	test := struct {
-		Code           *string  `json:"code,omitempty"`
-		Description    *string  `json:"description,omitempty"`
-		ExpectedValues []string `json:"expectedValues,omitempty"`
-		isValidated    bool
-	}{
-		ExpectedValues: sc.ExpectedValues,
-	}
+		Success     bool    `json:"success"`
+		Description *string `json:"description,omitempty"`
+		isValidated bool
+	}{}
 
-	// Empty strings are null
-	if sc.Code != "" {
-		test.Code = &sc.Code
+	if sc.Success != false {
```

Comment:
While this code technically works, it isn't correct.  You should always assign `test.Success = sc.Success` regardless of the sc.Success value.